.github/ISSUE_TEMPLATE/sip.md

```diff
@@ -6,9 +6,9 @@ labels: "#SIP"
 ---
 
 *Please make sure you are familiar with the SIP process documented*
-(here)[https://github.com/apache/superset/issues/5602]
+(here)[https://github.com/apache/superset/issues/5602]. The SIP number should be the next increment from the latest SIP listed [here](https://github.com/apache/superset/issues?q=is%3Aissue+label%3Asip).
 
-## [SIP] Proposal for XXX
+## [SIP-\<number>] Proposal for <title>
```

Comment:
```suggestion
## [SIP-<number>] Proposal for <title>
```

Reply:
I needed to escape this for the markdown to render correctly. I can reexamine this.

Reply:
oh, interesting. no worries then